modules/ROOT/pages/authentication-authorization/manage-users.adoc

```diff
@@ -581,6 +671,16 @@ RETURN user AS adminUser
 ----
 ======
 
+.Show user with auth
```

Comment:
separate from this PR as such but I do find it odd that we have two 'examples' above showing their results and then the parts that are actually rendered as examples don't have any results

Reply:
done